aws/resource_aws_eks_node_group.go

```diff
@@ -502,35 +503,35 @@ func resourceAwsEksNodeGroupUpdate(d *schema.ResourceData, meta interface{}) err
 		output, err := conn.UpdateNodegroupConfig(input)
 
 		if err != nil {
-			return fmt.Errorf("error updating EKS Node Group (%s) config: %w", d.Id(), err)
+			return diag.Errorf("error updating EKS Node Group (%s) config", d.Id(), err)
```

Comment:
Still need to pass `error`, just can't use `%w`.

```suggestion
			return diag.Errorf("error updating EKS Node Group (%s) config: %s", d.Id(), err)
```